components/Navigation.js

```diff
@@ -92,46 +92,51 @@ export default function Navigation(props) {
       <p>Logo will go here</p>
       <div className={classes.toolbar} />
       <Divider />
-      <List 
-      component="nav"
-      aria-labelledby="nested-list-subheader"
-      className={classes.root}
-    >
+      <List
+        component="nav"
+        aria-labelledby="nested-list-subheader"
+        className={classes.root}
+      >
         {navNodes.map((node) => {
           // console.log(node.label)
-          var isSubMenItem = (node.label === 'Gallery')
-          var subMListItem = (node.label === 'Travel' | node.label === 'Videography')
+          var isSubMenItem = node.label === 'Gallery';
+          var subMListItem =
+            (node.label === 'Travel') | (node.label === 'Videography');
```

Comment:
|| not |

Reply:
This is @daveGrayMac attempt at submenus, not going to change it on this PR, will open a new one for Navigation once shopping cart is done.

Reply:
Soz, just had to 😂